### PR TITLE
Upload service

### PR DESCRIPTION
**Pull request by Erlang233**
Add a defer close() statement to ensure the database connection will be closed, regardless of how the code executes.

### CODE REVIEW

**Review by jwagenaar — APPROVED** [review]
looks good to me

File: pkg/store/manifestFile.go [hunk 1]
```diff
@@ -365,47 +368,56 @@ func (s *manifestFileStore) SyncResponseStatusUpdate2(manifestId int32, failedFi
...
-	// format all vals at once
-	_, err := s.db.Exec(sqlStatement)
+	defer stmt.Close()
+
+	_, err := stmt.Exec(initiatedStatusStr, currentTime.Unix(), manifestId)
 	if err != nil {
-		log.Error(err)
-		return err
+		log.Println("the error is here:ResetStatusForManifest in second if statement" + "here is why" + error.Error())
+		return error
```
Comment (rohanshah) [inline pkg/store/manifestFile.go]: should these still reference `err` not `error`?

File: pkg/store/usersettings.go [hunk 2]
```diff
@@ -62,13 +63,19 @@ func (s *userSettingsStore) Get() (*UserSettings, error) {
...
-	statement, _ := s.db.Prepare("INSERT INTO user_settings (user_id, profile) VALUES (?, ?)")
-	_, err := statement.Exec(data.UserId, data.Profile)
+	statement, err := s.db.Prepare("INSERT INTO user_settings (user_id, profile) VALUES (?, ?)")
 	if err != nil {
-		log.Error("Unable to create user_record", err.Error())
+		log.Error("Error preparing statement for creating user settings:", err)
 		return nil, err
 	}
-
+	defer statement.Close() //
```
Comment (rohanshah) [inline pkg/store/usersettings.go]: 🧶 extra `//` can be removed here

**Review by rohanshah — COMMENTED** [review]
very nice cleanup! just left one question and one small nit comment

File: pkg/store/manifestFile.go (hunk 1 above)
Comment (Erlang233) [reply pkg/store/manifestFile.go]: yes, I fixed this typo in the newest version

File: pkg/store/usersettings.go (hunk 2 above)
Comment (Erlang233) [reply pkg/store/usersettings.go]: got it!

**Review by rohanshah — APPROVED** [review]

**Pull request merged**